object.c

```diff
@@ -4247,72 +4240,72 @@ f_sprintf(int c, const VALUE *v, VALUE _)
  *
  *  === Querying
  *
- *  - #!~:: Returns +true+ if +self+ does not match the given object,
- *          otherwise +false+.
- *  - #<=>:: Returns 0 if +self+ and the given object +object+ are the same
- *           object, or if <tt>self == object</tt>; otherwise returns +nil+.
- *  - #===:: Implements case equality, effectively the same as calling #==.
- *  - #eql?:: Implements hash equality, effectively the same as calling #==.
- *  - #kind_of? (aliased as #is_a?):: Returns whether given argument is an ancestor
- *                                    of the singleton class of +self+.
- *  - #instance_of?:: Returns whether +self+ is an instance of the given class.
- *  - #instance_variable_defined?:: Returns whether the given instance variable
- *                                  is defined in +self+.
- *  - #method:: Returns the Method object for the given method in +self+.
- *  - #methods:: Returns an array of symbol names of public and protected methods
- *               in +self+.
- *  - #nil?:: Returns +false+. (Only +nil+ responds +true+ to method <tt>nil?</tt>.)
- *  - #object_id:: Returns an integer corresponding to +self+ that is unique
- *                 for the current process
- *  - #private_methods:: Returns an array of the symbol names
- *                       of the private methods in +self+.
- *  - #protected_methods:: Returns an array of the symbol names
- *                         of the protected methods in +self+.
- *  - #public_method:: Returns the Method object for the given public method in +self+.
- *  - #public_methods:: Returns an array of the symbol names
- *                      of the public methods in +self+.
- *  - #respond_to?:: Returns whether +self+ responds to the given method.
- *  - #singleton_class:: Returns the singleton class of +self+.
- *  - #singleton_method:: Returns the Method object for the given singleton method
- *                        in +self+.
- *  - #singleton_methods:: Returns an array of the symbol names
- *                         of the singleton methods in +self+.
- *
- *  - #define_singleton_method:: Defines a singleton method in +self+
- *                               for the given symbol method-name and block or proc.
- *  - #extend:: Includes the given modules in the singleton class of +self+.
- *  - #public_send:: Calls the given public method in +self+ with the given argument.
- *  - #send:: Calls the given method in +self+ with the given argument.
+ *  - {!~}[#method-i-21~]: Returns +true+ if +self+ does not match the given object,
+ *    otherwise +false+.
+ *  - {<=>}[#method-i-3C-3D-3E]: Returns 0 if +self+ and the given object +object+ are the same
+ *    object, or if <tt>self == object</tt>; otherwise returns +nil+.
```

Comment:
```suggestion
 *  - #!~: Returns +true+ if +self+ does not match the given object,
 *    otherwise +false+.
 *  - #<=>: Returns 0 if +self+ and the given object +object+ are the same
 *    object, or if <tt>self == object</tt>; otherwise returns +nil+.
```